### ISSUE TITLE
URN identifier

### ISSUE THREAD

**Issue by Intelligent2013** [metanorma/metanorma-iso#658]
Source issue: https://github.com/metanorma/metanorma-iso/issues/648

Source adoc: https://github.com/metanorma/iso-iec-smart-terms

MN xml should provide URN identifier.

**Comment by ronaldtse** [comment]
Metanorma XML should provide the URN. Relaton ISO generates the correct URN.

**Comment by opoudjis** [comment]
This will come from pubid-iso

**Comment by ronaldtse** [comment]
Yes now pubid-iso provides the URN, but it has to be inserted inside Metanorma XML.
